### PR TITLE
[REVIEW]Fixed cudf.Series.to_gpu_array() with fillna param

### PR DESCRIPTION
**Pull request by AK-ayush**
This PR adds the feature of accepting the scalar values while creating the host array or gpu array from series. Minor code changes are done in `column.to_dense_buffer()` method. 
Once merged to master, issue #1048 can be closed.

### CODE REVIEW

**Comment by thomcom** [comment]
rerun tests

File: python/cudf/dataframe/column.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def to_gpu_array(self, fillna=None):
 
         Parameters
         ----------
-        fillna : str or None
+        fillna : str or None or scalar
```
Comment (thomcom) [inline python/cudf/dataframe/column.py]: Should be changed to `scalar, 'pandas', or None` - `str` is not actually accepted.

File: python/cudf/dataframe/column.py [hunk 2]
```diff
@@ -297,7 +297,7 @@ def to_array(self, fillna=None):
...
```
Comment (thomcom) [inline python/cudf/dataframe/column.py]: Same as above

File: python/cudf/dataframe/column.py [hunk 3]
```diff
@@ -482,7 +482,7 @@ def to_dense_buffer(self, fillna=None):
...
```
Comment (thomcom) [inline python/cudf/dataframe/column.py]: Same as above

File: python/cudf/dataframe/column.py (hunk 1 above)
Comment (AK-ayush) [reply python/cudf/dataframe/column.py]: sure doing it right now

**Pull request merged**